generator/src/test/java/com/scottlogic/deg/generator/generation/TestGenerationConfigSource.java

```diff
@@ -94,6 +94,10 @@ public File getProfileFile() {
     public boolean shouldViolate() {
         return false;
     }
```

Comment:
whitespace

Reply:
are you saying it needs more whitespace? less whitespace?

Reply:
looks fine now